packages/yargs/lib/logger.js

```diff
@@ -47,13 +47,20 @@ module.exports = exports = class Logger {
       console.log(`${prefix} ${message}`);
     }
   }
-  request(message) {
+  verbose(type, message) {
```

Comment:
I'm not too happy with this method having a different signature than the other methods on this class.

Reply:
Understood, will tackle this later in another PR. Specifically, I will replace the `verbose` method with something like this:

```javascript
exports.createLogger = (...args) =>
  new Proxy(new Logger(...args), {
    get(logger, prop) {
      return prop in logger
        ? logger[prop]
        : (message) => {
            const { level, name } = logger;
            const prefix = colorize(`${name}:${prop}`, 'gray');
            if (level >= logLevels.verbose) {
              console.log(`${prefix} ${message}`);
            }
          };
    },
  });
```

Anyhow, since there is `the-doctor` branch based on this one, I will not tackle this right away.

Reply:
This is the actual implementation in #304:

https://github.com/untool/untool/blob/c37129645fdd06e5ab53301c62bca4719890e8c6/packages/info/lib/logger.js#L61-L79